packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -0,0 +1,243 @@
+  storybookUrl="/?path=/story/components-chat-composer--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+## Guidelines
+
+### About Chat Composer
+
+A Chat Composer is an input made for users to type rich chat messages. Chat Composer is best used as one part of larger chat user interface to provide a seamless authoring experience.
+Within the context of Paste, Chat Composer is most typically used alongside the [`Chat Log`](/components/chat-log) component.
+
+### Accessibility
+
+Chat Composer supports a variety of aria attributes which are passed into the content editable region of the component. It is strongly encouraged to use an `aria-label` to describe the
+input itself.
+
+### Lexical and Plugins
+
+Chat Composer is built ontop of the [Lexical](https://lexical.dev) editor. Lexical is extensible and follows a declarative approach to configuration via JSX. Developers can leverage a
+wide variety of existing plugins or write their own to augment the behavior of Chat Composer. Plugins are provided via the `children` prop. As an example,
+Chat Composer uses a custom plugin [`AutoLinkPlugin`](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/chat-composer/src/AutoLinkPlugin.tsx) internally
+which you can see being configured [here](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/chat-composer/src/ChatComposer.tsx#L116) as a JSX child.
+
+## Examples
+
+### Basic
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Placeholder
+
+Set a placeholder value using a `placeholder` prop.
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Initial Value
+
+Set an initial value using an `initialValue` prop.
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} initialValue="This is my initial value" placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Max Height
+
+Restrict the height of the composer using a `maxHeight` prop.
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} maxHeight="size10" ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With ChatLog
```

Comment:
suggestion: naming this after the component instead of the React component would be nice since this example would also apply to Figma

```suggestion
### With Chat Log
```